lib/service/services_cli.rb

```diff
@@ -179,6 +179,7 @@ def kill(targets, verbose: false)
     # protections to avoid users editing root services
     def take_root_ownership(service)
       return unless System.root?
+      return true if System.systemctl?
```

Comment:
Why do this?

Reply:
Because Linux does not have Plist, so this https://github.com/Homebrew/homebrew-services/blob/ac860d1092dac7f270d8e8024ff9a8712b27a443/lib/service/services_cli.rb#L191 will return nil, causing the code following this https://github.com/Homebrew/homebrew-services/blob/ac860d1092dac7f270d8e8024ff9a8712b27a443/lib/service/services_cli.rb#L109 to be bypassed. But I have rewritten the code of `take_root_ownership`.